ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/export/export.tsx

```diff
@@ -67,6 +67,9 @@ export const getExportResult = (result: any) => {
   return {
     id: getResultId(result),
     source: getResultSourceId(result),
+    attributes: Object.keys(
+      result.attributes.metacard.attributes.properties.attributes
```

Comment:
✏️ use `.get()` to access a single attribute from a backbone model (for example line 58)